backend/migrations/20240214154028_food_env_score.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Add up migration script here
+
+CREATE DOMAIN env_rating as smallint CHECK (VALUE > 0);
```

Comment:
why do you use `smallint`s here?
In Rust you use `u32` which is a sensible default for an unsigned value, but smallints are only 2 bytes[1]. I think integer does fine here.

[1] https://www.postgresql.org/docs/current/datatype-numeric.html